lib/resources/utils.dart

```diff
@@ -62,6 +63,18 @@ class Utils {
 
   static PackageInfo packageInfo = PackageInfo(
       version: 'Unknown', buildNumber: 'Unknown', appName: 'Unknown');
+
+  static bool feedbackEnabled = false;
+
+  static Future<RemoteConfigService> getRemoteConfig() async {
+    try {
+      final remoteConfig = await RemoteConfigService.getInstance();
+      return remoteConfig;
+    } catch (e) {
+      AppToast.show(S.current.errorSomethingWentWrong);
```

Comment:
You're overusing this toast. You know you should try to show an actual error message, right? At least print it, you don't have to show it in the UI. You of all people should know it's hard to debug if everything just prints "Something went wrong".